crates/mun_hir/src/diagnostics.rs

```diff
@@ -178,6 +178,26 @@ impl Diagnostic for ExpectedFunction {
     }
 }
 
+#[derive(Debug)]
+pub struct ExportedPrivate {
+    pub file: FileId,
+    pub type_ref: AstPtr<ast::TypeRef>,
+}
+
+impl Diagnostic for ExportedPrivate {
+    fn message(&self) -> String {
+        "can't leak private type".to_string()
```

Comment:
We have duplication of this error message. Is there a way to reuse it to avoid code+paste errors?

Other occurance:
https://github.com/mun-lang/mun/pull/338/files#diff-7a57632f7acc9a0b1abad5ef6f9447f95fabecf03312d0ab7d0597807ae8e25dR24